Tribler/Core/Modules/restapi/events_endpoint.py

```diff
@@ -183,6 +185,13 @@ def on_circuit_removed(self, subject, changetype, circuit, *args):
             }
             self.write_data({"type": "circuit_removed", "event": event})
 
+    def on_search_response(self, subject, changetype, circuit, *args):
+        event = {
+            "search_query": args[0],
```

Comment:
I would just name this `query`.

Reply:
removed query from event, results should be sufficient